logstash-core-event-java/src/main/java/org/logstash/Event.java

```diff
@@ -338,22 +348,33 @@ private Timestamp initTimestamp(Object o) {
             logger.warn("Error parsing " + TIMESTAMP + " string value=" + o.toString());
         }
 
-        tag(TIMESTAMP_FAILURE_TAG);
-        this.data.put(TIMESTAMP_FAILURE_FIELD, o);
-
-        return Timestamp.now();
+        return null;
     }
 
     public void tag(String tag) {
-        List<Object> tags = (List<Object>) this.data.get("tags");
-        if (tags == null) {
+        List<Object> tags;
+        Object _tags = this.getField("tags");
```

Comment:
We are sweating very small stuff here. I thought we did have this discussion where non-fundamental or ad-hoc style choices, mostly based on personal preferences should be left at the author discretion. I don't see how this sets any styling precedent in any way. this is a local variable in a very local context. Regardless of Oracle style guide there are lots of precedents for using `_var` style of naming, typically for private vars. There is definitely no consensus around that, just like many other styling issues. At some point we have to respect some of the author decision. Obviously if the code is a mess, then it must be fixed but is this what we are talking about here?

Also, if we are going to use ES as an example, I urge you to execute this agaist the ES codebase and see for yourself.

```sh
ack "\s_[a-z][^\s]"  --type=java
```